interpretation/deepseismic_interpretation/azureml_pipelines/README.md

```diff
@@ -0,0 +1,174 @@
+# Integrating with AzureML
+
+## Running a Pipeline in AzureML
+Set the following environment variables:
+```
+BLOB_ACCOUNT_NAME
+BLOB_CONTAINER_NAME
+BLOB_ACCOUNT_KEY
+BLOB_SUB_ID
+AML_COMPUTE_CLUSTER_NAME
+AML_COMPUTE_CLUSTER_MIN_NODES
+AML_COMPUTE_CLUSTER_MAX_NODES
+AML_COMPUTE_CLUSTER_SKU
+```
+
+On Linux:
+`export VARIABLE=value`
+These can be set automatically in VSCode in an .env file, run with `source .env` in Linux or made into a `.bat` file to easily run from the command line in Windows. You can ask a team member for a .env file configured for our development environment to save time.
+
+Create a .azureml/config.json file in the project's root directory that looks like so:
+```json
+{
+"subscription_id": "<subscription id>",
+"resource_group": "<resource group>",
+"workspace_name": "<workspace name>"
+}
+
+```
+
+## Training Pipeline
+Here's an example of a possible config file:
+```json
+{
+    "step1":
+    {
+        "type": "PythonScriptStep",
+        "name": "process all files step",
+        "script": "process_all_files.py",
+        "input_datareference_path": "",
+        "input_datareference_name": "raw_input_data",
+        "input_dataset_name": "raw_input_data",
+        "source_directory": "src/first_preprocess/",
+        "arguments": ["--remote_run",
+        "--input_path", "input/",
+        "--output_path", "normalized_data"],
+        "requirements": "src/first_preprocess/preprocess_requirements.txt",
+        "node_count": 1,
+        "processes_per_node": 1
+    },
+    "step2":
```

Comment:
I understand that these are steps, but what do they do? Sound like step1 is step_data preprocessing, then step2 is data_normalization, etc... maybe more verbose names could be better?

Reply:
@maxkazmsft great question, @MJZawacki @GeisaFaustino @olgaliak @annazietlow which steps are we going to be committing to deepseismic here? I don't think it makes sense to add steps that we are not committing to the repo even though it would be a good starting point for others to build off of. If my understanding is correct we are going to be contributing the custom paths so the only step other than the train step should be the prepare files step to create the splits files so that would mean removing the process all files step.

Reply:
OK, makes sense - is it hard to add a hello-world example that shows how to use the trained model?

Reply:
discussed - will add a single training step only for the scope of this PR

Reply:
Pointing to docs on AzureML pipeline steps in this commit: https://github.com/microsoft/seismic-deeplearning/pull/195/commits/8e5cc721ee55b8deb55ddd9d4c11a238853e82c5 Modifying the example in the README.md to only include the training step that we use in this commit: https://github.com/microsoft/seismic-deeplearning/pull/195/commits/da15de2b6b7eacb04dc79e80ca1f66811d74495e